src/OSPSuite.R/Domain/ConcurrentRunSimulationBatch.cs

```diff
@@ -23,9 +23,9 @@ public class ConcurrentRunSimulationBatch : IDisposable, IWithId
 
       internal int MissingBatchesCount => Math.Max(0, SimulationBatchRunValues.Count - _simulationBatches.Count);
 
-      public ConcurrentRunSimulationBatch(IModelCoreSimulation simulation, SimulationBatchOptions simulationBatchOptions)
+      public ConcurrentRunSimulationBatch(Simulation simulation, SimulationBatchOptions simulationBatchOptions)
```

Comment:
Why this change here?

Reply:
As far as I can tell, this is always called with R.Domain.Simulation, which isn't set up for cloning. In another spot, I need to access the property CoreSimulation to actually create a clone of it. So rather than cast, I think it's ok that we just use the object.

Reply:
maybe I just need a default constructor - hold on

Reply:
Yes, with a default constructor for R.Domain.Simulation the clone manager is happy.

Reply:
What about you don't create a clone then but just instantiate a new Simulation?? If there was not default constructor, there was probalbgy a reason

Reply:
so return new Simulation(_clonedModelCoreSimulation)

Reply:
If this goes back to IModelCoreSimulation, then there's no time during the batch creation where I can get access to the underlying CoreSimulation without a cast. I need either the default constructor and clone, or cast to R.Domain.Simulation to get access to the underlying CoreSimulation so I can clone it and then instantiate an R.Domain.Simulation.